v5/v5_with_library/v5_stable.ino

```diff
@@ -83,6 +88,7 @@ bool thingspeak_turn = false;
 #define UPDATE_THING_SPEAK_TH_AFTER 5
 #define ALLOW_CREATING_NEW_VARIABLE_FILE true
 #define CONFIG_FILE "/config.txt"
+const String relayControllerURL = "http://192.168.0.7/?minute";
```

Comment:
have you make it static?

Reply:
yes, I had assigned this IP to my esp8266.